OpenRA.Game/Input/IInputHandler.cs

```diff
@@ -19,9 +19,10 @@ public interface IInputHandler
 		void OnKeyInput(KeyInput input);
 		void OnMouseInput(MouseInput input);
 		void OnTextInput(string text);
+		void OnGestureInput(MouseInput input);
 	}
 
-	public enum MouseInputEvent { Down, Move, Up, Scroll }
+	public enum MouseInputEvent { Down, Move, Up, Scroll, Gesture, FingerDown, FingerUp }
```

Comment:
Would it make sense to split these new values to a `GestureInputEvent` and make a new struct, `GestureInput` similar to `MouseInput` with only the properties needed for this use case?

Reply:
Sounds reasonable to me.